### DISCUSSION TITLE
Problem updating new install

### DISCUSSION THREAD

**Discussion by John-L** [termux/termux-packages#16490] [Q&A] [answered]
I tried this, with an error result:
`$ pkg update >Termux-error#1.txt 2>&1

Checking availability of current mirror: ok

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Ign:1 https://dl.bintray.com/grimler/game-packages-24 games InRelease
Ign:2 https://dl.bintray.com/grimler/science-packages-24 science InRelease
Err:3 https://dl.bintray.com/grimler/game-packages-24 games Release
  404  Not Found
Err:4 https://dl.bintray.com/grimler/science-packages-24 science Release
  404  Not Found
Hit:5 https://grimler.se/termux-packages-24 stable InRelease
Reading package lists...
E: The repository 'https://dl.bintray.com/grimler/game-packages-24 games Release' does not have a Release file.
E: The repository 'https://dl.bintray.com/grimler/science-packages-24 science Release' does not have a Release file.`

Then I tried this, and also got an error result:
`$ apt edit-sources >Termux-error#2.txt 2>&1

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

E: Sub-process editor returned an error code (100)`

Finally, the termux-change-repo command didn't seem to improve the situation.

Thanks in advance for any insight you may be able to provide.

**Comment by agnostic-apollo** [comment, marked as answer]
https://github.com/termux/termux-app#Installation

https://github.com/termux/termux-app#google-play-store-deprecated
